web/bitwarden/hub.go

```diff
@@ -216,6 +264,75 @@ func buildNotification(e *realtime.Event, userID string) *notification {
 	return &msg
 }
 
+func buildFolderPayload(e *realtime.Event, userID string) map[string]interface{} {
+	var updatedAt interface{}
+	var date string
+	if doc, ok := e.Doc.(*couchdb.JSONDoc); ok {
```

Comment:
Realtime events can contain documents in another format than `realtime.JSONDoc`?

Reply:
Yes, it's a bit of technical debt in the realtime, but it's hard to fix. The documents in realtime must match the `realtime.Doc` interface, and, in practice, it depends on how the update was made, and if we are using memory (self-hosted and development image) or redis (production with several cozy-stack servers).

Reply:
If we have an interface then could we use functions to access those attributes in a standardized fashion?

Reply:
Well, I don't see a convenient way to do that